src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,29 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { Movie } from '../MovieCard/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  // <>Put the list here</>
+  <>
+    <div className="movies">
+      {movies.map(film => (
+        <ul>
```

Comment:
do map inside of ul, because now you create a separate ul tag for each list item
![image](https://user-images.githubusercontent.com/57598825/147642782-2282e1f7-706e-45c1-9e47-91f6928d5a7f.png)